auditbeat/module/file_integrity/schema.fbs

```diff
@@ -31,6 +31,8 @@ table Metadata {
   mtime_ns:long;
   ctime_ns:long;
   type:Type = 1;
+  setuid:bool;
```

Comment:
Could we persist these bits with the `mode` to save a little space? I think currently we are at ~152 bytes per record.

Reply:
sure thing